cci-config-generator.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
Where did this come from? Do we need it checked in? I was assuming we'd switch over to circle 2.0 and get rid of the old config and never have to convert again...